<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>result</class>
 <widget class="QDialog" name="result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(230, 255, 139);
}
</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Indie Flower</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font:  bold 30px &quot;Indie Flower&quot;;
color: black
}</string>
      </property>
      <property name="text">
       <string>RESULT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label1">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string>Набрано символов: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label2">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string>Ошибок:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label3">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string>Время:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label4">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string>Скорость:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="symbolsLabel">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errsLabel">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="speedLabel">
          <property name="font">
           <font>
            <family>Georgia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
color: black
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="font">
       <font>
        <family>Indie Flower</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: rgb(191, 191, 191);
font:  18px &quot;Indie Flower&quot;;
color: black
}</string>
      </property>
      <property name="text">
       <string>GREAT!</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
